src/GrumPHP/Task/ComposerScript.php

```diff
@@ -62,6 +64,7 @@ public function run(ContextInterface $context)
         $arguments->addRequiredArgument('%s', $config['script']);
 
         $process = $this->processBuilder->buildProcess($arguments);
+        $process->setWorkingDirectory($config['working_directory']);
```

Comment:
The run-script command actually has a working dir option.
Maybe it's better to use that one if the working_directory is set and default it to null?

```
  -d, --working-dir=WORKING-DIR  If specified, use the given directory as working directory.
```